doc/source/docsportal_sec.rst.inc

```diff
@@ -0,0 +1,199 @@
+    required checks and merge content manually
+  * Once all the prerequisites are fulfilled Zuul merges Pull
+    Request, builds documentation and pushes it to storage with
+    dedicated credentials.
+  * Only git projects explicitly included in the Zuul tenant are
+    being respected. Registered git projects with disabled
+    branch protection rules are ignored.
+
+* `GitHub <https://github.com>`_
+
+  * An external git hosting provider.
+  * Projects in the GitHub organization are managed by `dedicate process
+    <https://docs.otc-service.com/system-config/gitcontrol.html>`_
+
+Interface Description
+---------------------
+
+The only public facing interface is the regular Web using HTTPS (automatic
+forwarding from HTTP).
+
+Tenant Security
+---------------
+
+Documentation Portal is not providing any tenant support. All documents that are hosted
+on the Help Center are placed in a dedicated storage (as public content). User
+management is also not supported.
+
+O&M Access Control
+------------------
+
+Only users enabled in the :git_file:`inventory/base/group_vars/all.yaml` are
+able to login to the underlaying infrastructure. Direct access to the hosts is
+only possible through the :ref:`Bridge` host which serves as a bastion host.
+
+Logging and Monitoring
+----------------------
+
+Every component of the HelpCenter produces own logs.
+
+* haproxy log (VM service logs)
+* nginx log (VM or Kubernetes POD log)
+* Swift proxy and storage service logs
+* Zuul logs
+
+  * public job logs (test build log file)
+  * executor log
+  * scheduler log
+
+Patch Management
+----------------
+
+The service consists of OpenSource elements only. Whenever new release of any
+software element (haproxy, nginx, zuul) is identified a Pull Request to this
+repository need to be created to update the software. Pathing of the
+underlaying VM (haproxy) is executed as a regular job applying all the existing
+OS updates.
+
+Hardening
+---------
+
+All configuration files for the hosts, Cloud Load Balancer configuration and K8
+configuration is part of this repository. Every VM is managed by the System
+Config project applying the same hardening rules to evenry host according to
+the configuration.
+
+Web server is deployed as a Kubernetes pod in a dedicated namespace and receives an Ingress.
+
+Certificate Handling
+--------------------
+
+SSL Certificates are obtained using Let's Encrypt Certificate authority.
+Following is important:
+
+* Certificate for the K8 deployment can managed by the
+  `CertManager <https://cert-manager.io/>`_ deployed in
+  the Kubernetes cluster. This is achieved by placing
+  Kubernetes annotation on the deployment.
+* Alternatively SSL Certificate for the K8 installation may be generated on the
+  deployment server and provided into the K8 as secrets.
+* Certificates for the other involved components (Zuul,
+  Swift) are managed by the corresponding components
+  themselves.
+
+Backup and Restore
```

Comment:
basically you can mention that all data are stored in their source form in github with all the history modifications as well.
So it's simple to restore documentation to specific time in past if needed by rebuilding whole documentation to that point.

Reply:
done